octavia-cli/integration_tests/conftest.py

```diff
@@ -124,3 +127,73 @@ def connection(api_client, workspace_id, octavia_test_project_directory, source,
     yield connection
     connection.api_instance.delete_connection(connection.resource_id_request_body)
     silent_remove(configuration_path)
+
+
+@pytest.fixture(scope="session")
+def secured_api_client():
+    return get_api_client("http://localhost:8010", api_headers=[
+        ApplicationHeader(name="Authorization", value="Basic dXNlcjE6cGFzc3dvcmQ=")
```

Comment:
Please perform the base 64 encoding in python as it will make it easier to understand this is the value of the user:password you've set in `nginx.conf`